docs/tempo/website/metrics-generator/app-performance-mgmt.md

```diff
@@ -0,0 +1,168 @@
+
+Application performance management (APM) uses data gathered from services, agents, systems, and microservices to visualize and identify areas of concern.
+
+The Grafana APM dashboard visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
+Once the requirements are set up, this pre-configured dashboard is immediately available.  
+
+You can use the APM dashboard to:
+
+* Discover spans which are consistently erroring and the rates at which they occur
+* Get an overview of the overall rate of span calls throughout your services
+* Determine how long the slowest queries in your service take to complete
+* Examine all traces that contain spans of particular interest based on rate, error and duration values (RED signals)
+
+<p align="center"><img src="../../getting-started/apm-overview.png" alt="APM dashboard"></p>
+
+## Requirements to enable the APM dashboard
+
+You have to enable span metrics and service graph generation on the Grafana backend so metrics that are generated as traces are ingested.
+
+To use the APM dashboard, you need:
+
+* Tempo or Grafana Cloud Traces with either 1) the metrics generator enabled and configured or 2) the Grafana Agent enabled and configured to send data to a Prometheus-compatible metrics store
+* [Services graphs]({{< relref "../metrics-generator/service_graphs/#how-to-run" >}}), which are enabled by default in Grafana
+* [APM table enabled](https://grafana.com/docs/grafana/next/datasources/tempo/) within the service graph tab (use the `tempoApmTable` feature flag in Grafana to enable)
+* [Span metrics]({{< relref "../metrics-generator/span_metrics/#how-to-run" >}}) enabled in your Tempo data source configuration
+
+The APM dashboard can be generated with the metrics-generator or Grafana Agent.
+
+For information on how to configure these features, refer to the [Grafana Tempo data sources documentation](https://grafana.com/docs/grafana/next/datasources/tempo/).
+
+## APM dashboard
+
+Using this dashboard, you can see the top five spans by request rate. You can refine any of this data using the filters. Selecting any of the data points lets you see more specific data.
+
+The APM dashboard provides a span metrics visualization (APM table, screen section 2) and service graph (screen section 3). In addition, you can use the filters (screen section 1) to customize the data displayed.
+
+<p align="center"><img src="../apm-overview-numbered.png" alt="APM dashboard with numbered sections"></p>
+
+Any information in the APM table that has an underline can be selected to show more detailed information. You can also select any node in the service graph to display additional information. In the dashboard shown below, the `Ingester.QueryStream` span has a request rate of `144220.22` requests per second. The `cortex/IngesterQuery` span has the highest request rate.
```

Comment:
cortex -> mimir?

Reply:
Good catch. I hadn't realized the text didn't match the screen shot. Fixed that.